release-notes/opensearch.job-scheduler.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,23 @@
+## Version 1.2.0.0 2021-11-05
+
+Compatible with OpenSearch 1.2.0
+
+### Features
+
+* Adds a delay parameter to the job scheduler ([#61](https://github.com/opensearch-project/job-scheduler/pull/61))
```

Comment:
This was part of 1.1 already

Reply:
It happened as patch after `1.1.0.0` so it's not going to be released until OS `1.1.1.0`. I've added it here since this will be the first release with the change. It should also be included in the `1.1.1.0` release notes since that's an addition to 1.1 that should be called out I think.